Chocotastic/Views/CartCell.swift

```diff
@@ -0,0 +1,83 @@
+/// Copyright (c) 2023 Razeware LLC
+/// 
+/// Permission is hereby granted, free of charge, to any person obtaining a copy
+/// of this software and associated documentation files (the "Software"), to deal
+/// in the Software without restriction, including without limitation the rights
+/// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+/// copies of the Software, and to permit persons to whom the Software is
+/// furnished to do so, subject to the following conditions:
+/// 
+/// The above copyright notice and this permission notice shall be included in
+/// all copies or substantial portions of the Software.
+/// 
+/// Notwithstanding the foregoing, you may not use, copy, modify, merge, publish,
+/// distribute, sublicense, create a derivative work, and/or sell copies of the
+/// Software in any work that is designed, intended, or marketed for pedagogical or
+/// instructional purposes related to programming, coding, application development,
+/// or information technology.  Permission for such use, copying, modification,
+/// merger, publication, distribution, sublicensing, creation of derivative works,
+/// or sale is expressly withheld.
+/// 
+/// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+/// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+/// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+/// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+/// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+/// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+/// THE SOFTWARE.
+
+import UIKit
+import RxSwift
+
+class CartCell: UITableViewCell {
+  static let Identifier = "CartCell"
+  private var disposeBag = DisposeBag()
+  private var chocolate: Chocolate!
+  private var cart: ShoppingCart!
+  
+  @IBOutlet weak var emojiLabel: UILabel!
+  @IBOutlet weak var countryNameLabel: UILabel!
+  @IBOutlet weak var countLabel: UILabel!
+  @IBOutlet weak var minusButton: UIButton!
+  @IBOutlet weak var plusButton: UIButton!
+  
+  override func prepareForReuse() {
+    super.prepareForReuse()
+    disposeBag = DisposeBag()
+  }
+  
+  func configure(with chocolate: Chocolate, cart: ShoppingCart) {
+    self.chocolate = chocolate
+    self.cart = cart
+    
+    emojiLabel.text = "🍫" + chocolate.countryFlagEmoji
+    countryNameLabel.text = chocolate.countryName
+    
+    // チョコレートの数をcountLabelにバインドする
+    cart.chocolates
+      .map { chocolates in chocolates.filter { $0 == chocolate }.count }
```

Comment:
同じオペレーターでの処理は一つにまとめることが多いです。

```suggestion
      .map { String(chocolates in chocolates.filter { $0 == chocolate }.count) }
```